### ISSUE TITLE
Upgrade copypasta to use x11-clipboard 0.4

### ISSUE THREAD

**Issue by alberdingk-thijm** [jwilm/alacritty#3018]
A PR was recently merged into `x11-clipboard` (https://github.com/quininer/x11-clipboard/pull/17) to fix some build errors due to [how docs.rs changed in September](https://blog.rust-lang.org/2019/09/18/upcoming-docsrs-changes.html).

Can copypasta bump its version of x11-clipboard to 0.4 so that this is fixed?
While copypasta's docs work fine, other dependent crates like `conrod_core` are having [build errors](https://docs.rs/crate/conrod_core/0.68.0/builds/195775) due to this dependency.

**Comment by kchibisov** [comment]
Sure, please, send a PR.

**Comment by chrisduerr** [comment]
As @kchibisov stated, PRs in this regard are very welcome and of course always take a tiny bit of load off of Alacritty maintainers, so feel free to just send one.

Otherwise I'll create one myself later, with this issue it is of course trivial.

**Comment by chrisduerr** [comment]
Also @alberdingk-thijm, just to clarify, you'd probably require a new release of the `copypasta` depedency, right? I can push one once the PR from cole is merged.

**Comment by chrisduerr** [comment]
Will release a new version as soon as https://github.com/jwilm/alacritty/pull/3024 is merged.

**Comment by chrisduerr** [comment]
Version 0.6.1 has been released. If there are any more issues, please let me know.
